gcloud-java-examples/src/main/java/com/google/gcloud/examples/bigquery/BigQueryExample.java

```diff
@@ -60,11 +60,10 @@
  *
  * <p>Steps needed for running the example:
  * <ol>
- * <li>login using gcloud SDK - {@code gcloud auth login}.</li>
- * <li>compile using maven - {@code mvn compile}</li>
- * <li>run using maven -
- * <pre>{@code mvn exec:java -Dexec.mainClass="com.google.gcloud.examples.bigquery.BigQueryExample"
- *  -Dexec.args="[<project_id>]
+ * <li>log in using gcloud SDK - {@code gcloud auth login}.</li>
+ * <li>compile using maven - {@code cd gcloud-java-examples; mvn package appassembler:assemble -DskipTests -Dmaven.javadoc.skip=true -Dmaven.source.skip=true}</li>
```

Comment:
Using:
```
 * <li>compile using maven - {@code cd gcloud-java-examples;
 * mvn package appassembler:assemble -DskipTests -Dmaven.javadoc.skip=true -Dmaven.source.skip=true}
 * </li>
```
Should not affect the resulting html while remaining within the 100 characters limit.
This comment also applies to other files